### ISSUE TITLE
Learning Constrained Adaptive Differentiable Predictive Control Policies With Guarantees. (arXiv:2004.11184v6 [eess.SY] UPDATED)

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#15213]
Learning Constrained Adaptive Differentiable Predictive Control Policies With Guarantees. (arXiv:2004.11184v6 [eess.SY] UPDATED)<br>
https://ift.tt/3cAwJFU<br>
<p>We present differentiable predictive control (DPC), a method for learning constrained neural control policies for linear systems with probabilistic performance guarantees. We employ automatic differentiation to obtain direct policy gradients by backpropagating the model predictive control (MPC) loss function and constraints penalties through a differentiable closed-loop system dynamics model. We demonstrate that the proposed method can learn parametric constrained control policies to stabilize systems with unstable dynamics, track time-varying references, and satisfy nonlinear state and input constraints. In contrast with imitation learning-based approaches, our method does not depend on a supervisory controller. Most importantly, we demonstrate that, without losing performance, our method is scalable and computationally more efficient than implicit, explicit, and approximate MPC.</p>
<p>Under review at IEEE Transactions on Automatic Control.</p>
<br>
<br>
via eess.SY updates on arXiv.org http://arxiv.org/<br>
